contracts/operators/CollectionBatchBuyOperator.sol

```diff
@@ -109,17 +116,16 @@ contract CollectionBatchBuyOperator is IOperator {
         }
 
         // Lengths of arrays are updated at the end.
-        uint256[] memory tokenIds = new uint256[](ex.numOfTokens);
+        uint256[] memory purchasedTokenIds = new uint256[](ex.numOfTokens);
 
         // Get the expected receiver of the tokens.
         address receiver = ex.isReceivedDirectly ? msg.sender : address(this);
-
-        uint96 totalEthUsed;
-        uint256 tokensBought;
+        uint256 numTokensBought;
         for (uint256 i; i < ex.calls.length; ++i) {
             BuyCall memory call = ex.calls[i];
 
-            uint96 callValue;
+            uint96 callValue = 0;
```

Comment:
For style/consistency, I like to avoid initializing vars to their default

Reply:
alright